### PR TITLE
*: use glide.

### PR DESCRIPTION
**Pull request by siddontang**
@qiuyesuifeng

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,10 +6,17 @@ build:
...
 	go test --race -timeout 2m ./...
 
 clean:
 	go clean -i ./...
 	@rm -rf ./bin
+
+update_vendor:
+	which glide >/dev/null || curl https://glide.sh/get | sh
+	which glide-vc || go get -v -u github.com/sgotti/glide-vc
+	glide --verbose update --strip-vendor
```
Comment (qiuyesuifeng) [inline Makefile]: I tried this command, but got an error `--strip-vendor cannot be used without --strip-vcs`
@overvenus any suggestion?

Comment (qiuyesuifeng) [inline Makefile]: My glide version 0.11.1

Comment (overvenus) [inline Makefile]: Try install dev version, `go get -u github.com/Masterminds/glide`

Comment (qiuyesuifeng) [inline Makefile]: My fault, i used homebrew for glide install, i will update it.

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**